frontend/src/pages/auth/auth.tsx

```diff
@@ -27,6 +40,13 @@ const Auth: React.FC = () => {
     [dispatch],
   );
 
+  useEffect(() => {
+    if (redirectTo) {
+      navigate(redirectTo);
+      dispatch(appActions.navigate(null));
+    }
+  }, [dispatch, navigate, redirectTo]);
```

Comment:
we don't need this here, we should dispatch navigate inside signUp, signIn thunkActions
```suggestion
```